gui_build.py

```diff
@@ -276,6 +276,9 @@ def update_command_display(self):
         # Can't select regression & tests
         #selected_tests = [test for test, var in self.tests_vars.items() if var.get()]
         selected_cfg = [cfgs.rsplit('.', 1)[0] for cfgs, var in self.cfg_vars.items() if var.get()]
+        selected_cfg_count = len(selected_cfg)
```

Comment:
This will work for now.
Please open a new issue that we can track.
It should say "refactor the -cfg to be a pull-down list"
Explain in the issue the correct state, and what should to be turned into (similar to -regress, -top)